### ISSUE TITLE
get list of channel members

### ISSUE THREAD

**Issue by luzlab** [microsoftgraph/microsoft-graph-docs#4925]
Is there a way to retrieve a list of channel members via the API?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 75f86208-723e-a0c9-2cce-3e681dfa7b9c
* Version Independent ID: dc38068e-3b39-1352-117d-e770485b7b70
* Content: [channel resource type - Microsoft Graph v1.0](https://docs.microsoft.com/en-us/graph/api/resources/channel?view=graph-rest-1.0)
* Content Source: [api-reference/v1.0/resources/channel.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/master/api-reference/v1.0/resources/channel.md)
* Product: **microsoft-teams**
* Technology: **microsoft-graph**
* GitHub Login: @nkramer
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by nkramer** [comment]
Channels don't have members, teams do.

**Comment by luzlab** [comment]
Let me rephrase the question - Is there a way to retrieve a list of _team members_ that have **favorited** a channel via the API?

**Comment by nkramer** [comment]
No.

**Comment by nkramer** [comment]
But please upvote that suggestion on UserVoice -- https://microsoftgraph.uservoice.com/forums/920506-microsoft-graph-feature-requests/suggestions/35129218-we-would-like-to-have-the-graph-call-for-joined-te

**Comment by tsteckenborn** [comment]
I'ld like to have this request reopened. With private channels it is now useful to have a member list, as well as a possibility to add members.

**Comment by carlo-quinonez** [comment]
The endpoints for this already exist, and I opened another issue #8469 to add this information to the documentation.
